### DISCUSSION TITLE
Delete all file in logs directory

### DISCUSSION THREAD

**Discussion by BitTheCat** [opcodesio/log-viewer#47] [Ideas]
Hi, I realized that the button to empty the log folder (thus total delete) is missing. Is it possible to implement it?

**Comment by vesper8** [comment]
Yes please.. this was one of my gripes with arcanedev/log-viewer too and they too never implemented the "clear all logs" feature.. I use this all the time.

It would also be nice to be able to configure a logrotate schedule as part of this UI

**Comment by susanBuck** [comment]
Yes please. Would love to see this feature added.

**Comment by BitTheCat** [comment]
@arukompas any news?

**Comment by arukompas** [comment]
Hey guys, I'll have more free time this friday so I'll try to come up with a nice and clean solution then! I know deleting multiple files isn't hard, but it's more of a UX consideration :)

**Comment by arukompas** [comment]
Hey everyone,

Folder actions have been added in `v1.5.0` - please update the Log Viewer in your app! 😄 

_cc @BitTheCat @susanBuck @vesper8_

**Reply by susanBuck** [reply]
You rock! Thanks.

**Reply by vesper8** [reply]
Thanks @arukompas 

So.. does deleting the root "logs" folder just delete all log files inside the logs folder.. or does it actually delete the logs folder itself? I guess it's re-created but if the folder is being deleted that's not ideal.. could cause errors if other processes, such as supervisor, tries to create log files inside that folder.

**Reply by arukompas** [reply]
Hey @vesper8 

It never deletes any folders. Just the files.
